core-common/src/main/java/eu/arrowhead/common/CommonConstants.java

```diff
@@ -252,10 +252,12 @@
 	public static final String OP_CONFIGURATION_MGMT_MANAGE = "/mgmt/config";
 
 	public static final String CHOREOGRAPHER_URI = "/choreographer";
-	public static final String OP_CHOREOGRAPHER_NOTIFY_STEP_DONE = "/executor/notifyStepDone";
+	public static final String CHOREOGRAPHER_EXECUTOR_MGMT_URI = "/executor" + CoreCommonConstants.MGMT_URI;
+	public static final String CHOREOGRAPHER_EXECUTOR_MGMT_BY_ID_URI = CHOREOGRAPHER_EXECUTOR_MGMT_URI + "{/id}";
 	public static final String OP_CHOREOGRAPHER_EXECUTOR_REGISTER = "/executor/register";
 	public static final String OP_CHOREOGRAPHER_EXECUTOR_UNREGISTER = "/executor/unregister";
-	public static final String OP_CHOREOGRAPHER_EXECUTOR_NOTIFY_STEP_ERROR = "executor/notifyStepError";
+	public static final String OP_CHOREOGRAPHER_NOTIFY_STEP_DONE = "/executor/notifyStepDone";
```

Comment:
according to the original code review of choreographer, this should be notify-step-done and notify-step-error